rita_client/src/dashboard/interfaces.rs

```diff
@@ -11,8 +11,8 @@ use crate::RitaClientError;
 
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct InterfaceToSet {
-    pub interface: String,
```

Comment:
Minor nit of no real value. struct name should probably also change to InterfacesToSet